fastlane/Fastfile

```diff
@@ -77,6 +78,14 @@ platform :ios do
     )
   end
 
+  desc "create git tag and push tag"
+  lane :create_git_tag do |options|
+    version = options[:version]
+
+    add_git_tag(tag: "#{version}")
+    push_git_tags
+  end
+
```

Comment:
add create git tag in fastfile steps

Reply:
Is the git tag number different from the version number?

Reply:
they should be the same. this step takes a `version` that we set, and then create a git tag that's labeled with the `version` number